docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -465,16 +465,22 @@ You can also cancel all pending tasks at once using the [runner.stop](#stop) fun
 
 #### Quarantine Mode
 
-The quarantine mode is designed to isolate *non-deterministic* tests (that is, tests that sometimes pass and fail without any apparent reason)
-from the rest of the test base (*healthy* tests).
+The quarantine mode is designed to isolate *non-deterministic* tests (that is, tests that pass and fail without any apparent reason) from the other tests.
 
-In this mode, a failed test is executed several times. The test result depends on the outcome (passed or failed) that occurs most often. That is, if the test fails on most attempts, the result is failed. If the test result differs between test runs, the test is marked as unstable.
+When the quarantine mode is enabled, tests run according to the following logic:
+
+1. A test runs at the first time. If it passes, TestCafe proceeds to the next test.
+2. If the test fails, it runs over and over until it shows the same result (passed or failed) three times.
+3. The result seen most often (three times) is recorded as the test result.
+4. If the test result differs between test runs, the test is marked as unstable.
+
+> When you decide whether to enable quarantine mode on your test machine, note that it increases test task duration because failed tests are executed three to five times.
```

Comment:
"When you decide whether to"?
-> Note that it increases the test task's duration if you enable quarantine mode on your test machine because failed tests are executed three to five times.